### ISSUE TITLE
expect-expect considers functions?

### ISSUE THREAD

**Issue by AntonioVentilii** [vitest-dev/eslint-plugin-vitest#696]
Honest doubt, is the following flagged by `expect-expect` rule? Or better, shall this one be flagged? Is there an option to not flag it?

```ts
export const testDerivedUpdates = async (...) => {
....
	derivedMocks.forEach((mockFn) => expect(mockFn).toHaveBeenCalledTimes(2));
....
};


it('should ......', async () => {
	await testDerivedUpdates(() =>
		initTokenActionValidationErrorsContext({
			errorType: 'insufficient-funds'
		})
	);
});
```

Ignore the logic per-se, it is quite more complex to the point that I cannot put the `expect` function directly in the test, but use another function.

**Comment by veritem** [comment]
you can always disable an eslint rule per file https://www.google.com/search?q=disable+eslint+by+file&oq=disable+eslint+by+file&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIGCAEQLhhA0gEIMzc2OGowajGoAgCwAgA&sourceid=chrome&ie=UTF-8

**Comment by AntonioVentilii** [comment]
@veritem thank you for the answer!

Should there maybe be an option to check into the functions called in the test? I imagine it is a common pattern to call functions that contain tests, so that there is no `expect` inside an `it(...)` statement.
